lib/SILOptimizer/Mandatory/TFConstExpr.cpp

```diff
@@ -439,7 +419,7 @@ namespace {
   /// callee in a call chain to represent the constant values given the set of
   /// formal parameters that callee was invoked with.
   class ConstExprFunctionCache {
```

Comment:
side note, not necessary to address in this PR: I think that the name "cache" is confusing. It feels more like a state machine than a cache. (`evaluateFlowSensitive` is the machine's transition function)